doc/usage.rst

```diff
@@ -375,6 +375,25 @@ keeping a decompressed copy of the file:
         data = numpy.load(fname)
         return data
 
+:class:`pooch.Decompress` returns ``"large-binary-file.npy.gz.decomp"`` as the
+decompressed file name by default. You can change its behaviour by passing a
+callback instead:
+
+.. code:: python
+
+    from pooch import Decompress
+
+    def fetch_compressed_file():
+        """
+        Load a large binary file that has been gzip compressed.
+        """
+        # Pass in the processor to decompress the file on download
+        fname = GOODBOY.fetch("large-binary-file.npy.gz",
+                              processor=Decompress(rename=lambda fname: fname[:-3]))
```

Comment:
Improve styling of docs and how we can remote the compress file extension, independently of the number of characters of the extension.

```suggestion
        # Pass in the processor to decompress the file on download
        fname = GOODBOY.fetch("
            large-binary-file.npy.gz",
            processor=Decompress(rename=lambda fname: os.path.splitext(fname)[0]),
        )
```